### ISSUE TITLE
download progress bar not displaying properly 

### ISSUE THREAD

**Issue by sinsoft76** [Modorganizer2/modorganizer#478]
# The problem

the download progress bar stopping at random progress then starting back up at a higher progress or completed. while not keeping track of progress until displaying again

Briefly describe the issue you are experiencing (or the feature you want to see added to Mod Organizer). Tell us what you were trying to do and what happened instead. Remember, this is _not_ a place to ask questions. For that, go to our discord channel

# Environment

mod organizer 2 2.1.4

* Mod Organizer Version that exhibits the issue:
* Last Mod Organizer Verison that did not exhibit the issue (if applicable):
* Desktop OS/version used to run Mod Organizer:

# Details

If necessary, describe the problem you have been experiencing in more detail.
every time I download anything using MO2 the progress bar stops at random and displays nothing till it decides to display again. not even the % downloaded shows till it starts back up again. 

# Link to Mod Organizer logs

## USVFS
Create a [GIST](https://gist.github.com) which is a paste of your USVFS logs, and link them here.
Do _NOT_ paste your full USVFS logs here, as it will make this issue very long and hard to read!
If you are reporting a bug, _always_ include the USVFS logs!

## MO Interface
Create a [GIST](https://gist.github.com) which is a paste of your mo_interface logs, and link them here.
Do _NOT_ paste your full mo_interface logs here, as it will make this issue very long and hard to read!
If you are reporting a bug, _always_ include the mo_interface logs!

**Comment by DankRafft** [comment]
Yeah, I noticed the display error with the newest version, too. When it stops displaying the correct progress and download rate just hover the mouse over the download in question and it should refresh.
